4-storage/redshift-storage/sql/org.schema/web_page_1.sql

```diff
@@ -40,4 +40,5 @@ CREATE TABLE atomic.org_schema_web_page_1 (
 DISTSTYLE KEY
 -- Optimized join to atomic.events
 DISTKEY (root_id)
-SORTKEY (root_tstamp);
+SORTKEY (root_tstamp)
+FOREIGN KEY(root_id) REFERENCES events(event_id);
```

Comment:
These FOREIGN KEY references are in the wrong place (they were copied from an erroneous schema I wrote). This is the right format: https://github.com/snowplow/snowplow/blob/master/4-storage/redshift-storage/sql/com.google.analytics/cookies_1.sql#L37